setup.py

```diff
@@ -59,8 +59,7 @@
         'pytest-metadata>=1.5.1',
         'pytest-mock',
         'pytest-xdist',
-        ('ESMValTool_sample_data @ '
-         'git+https://github.com/ESMValGroup/ESMValTool_sample_data@v0.0.3'),
+        'ESMValTool_sample_data>=v0.0.3',
```

Comment:
```suggestion
        'ESMValTool_sample_data==0.0.3',
```

Reply:
Let's keep the pin to a specific version, because we do not want the ESMValCore the unit tests might break as soon as we release a new version of the ESMValTool_sample_data.